src/cli/cli.cpp

```diff
@@ -2013,6 +2014,10 @@ void Interpreter::HandleIcmpReceive(otMessage *          aMessage,
     mServer->OutputFormat("\r\n");
 
 exit:
+    if (!mPingTimer.IsRunning())
```

Comment:
It is logical to print `Done` after `ping` have received ICMP replies. However there are some issues regarding to this behavior:
- `ping` is not guaranteed to receive ICMP echo reply, violating the rule that `CLI commands always output command result`
- There can be a long delay between `ping ...` command and `Done`. I run `ping count=5` and `scan` at the same time, got following output:
```
> ping fdde:ad00:beef:0:6926:9463:d7be:21e6 4 5 1 64
12 bytes from fdde:ad00:beef:0:6926:9463:d7be:21e6: icmp_seq=24 hlim=64 time=253ms
12 bytes from fdde:ad00:beef:0:6926:9463:d7be:21e6: icmp_seq=25 hlim=64 time=1ms
12 bytes from fdde:ad00:beef:0:6926:9463:d7be:21e6: icmp_seq=26 hlim=64 time=1ms
> scan
| J | Network Name     | Extended PAN     | PAN  | MAC Address      | Ch | dBm | LQI |
+---+------------------+------------------+------+------------------+----+-----+-----+
| 0 | OpenThread       | dead00beef00cafe | face | 221f8796816c1847 | 11 | -20 |   0 |
Done // scan
12 bytes from fdde:ad00:beef:0:6926:9463:d7be:21e6: icmp_seq=27 hlim=64 time=4067ms
Done // what is this?
12 bytes from fdde:ad00:beef:0:6926:9463:d7be:21e6: icmp_seq=28 hlim=64 time=3070ms
Done // ping
```
It becomes become difficult to track which `Done` is for which command, since commands and `Done`s are not one after another.  Also there are more than 2 `Done`s, should be a bug.

Reply:
> Also there are more than 2 Dones, should be another issue.

Should be due to that the pings are delayed by `scan`, causing node have receive multiple ping replies after `mPingTimer` stops.